server/index.js

```diff
@@ -18,20 +19,65 @@ app.get('/api/buytest', (req, res) => {
       res.send(err);
     } else {
       res.send(docs);
-    }    
+    }
   });
 });
 
-app.get('/api/bpowertest', (req, res) =>{
-  dpower.find((err, docs) => {
+app.get('/api/buytest/one/:id', (req, res) => {
+  const id = JSON.parse(req.params.id);
+  modalBuy.read(id, (err, doc) => {
```

Comment:
Related to the conversation we had with Kent earlier: Not necessarily, but might be helpful for future work to factor out these Mongoose queries to a separate file for better separation of concerns and more database modularity (so when we swap out the database later, you don't have to change the code in this file as much).

Reply:
Right. I like the idea. I think I'll use a routing page to connect to controllers.